guides/source/i18n.md

```diff
@@ -42,7 +42,7 @@ Internationalization is a complex problem. Natural languages differ in so many w
 
 As part of this solution, **every static string in the Rails framework** - e.g. Active Record validation messages, time and date formats - **has been internationalized**. _Localization_ of a Rails application means defining translated values for these strings in desired languages.
 
-To localize store and update _content_ in your application (e.g. translate blog posts), see the [Translating model content](#translating-model-content) section.
+To localize, store and update _content_ in your application (e.g. translate blog posts), see the [Translating model content](#translating-model-content) section.
```

Comment:
One more, since we [encourage Oxford commas](https://guides.rubyonrails.org/v7.0/api_documentation_guidelines.html#oxford-comma):

```suggestion
To localize, store, and update _content_ in your application (e.g. translate blog posts), see the [Translating model content](#translating-model-content) section.
```

Reply:
My bad, english is my second language. Thanks